internal/uvm/scsi.go

```diff
@@ -17,6 +18,14 @@ const (
 	lcowSCSILayerFmt = "/run/layers/S%d/%d"
 )
 
+type VMAccessType int
+
+const (
+	VMAccessTypeNoop = iota
```

Comment:
You can make these values be explicitly typed as `VMAccessType` by writing this as: `VMAccessTypeNoop VMAccessType = iota`.